tests/conformance_suite.postman_collection.json

```diff
@@ -6797,6 +6797,81 @@
 							},
 							"response": []
 						},
+						{
+							"name": "credentials_issue:credential.opt.id",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.test(\"status code is 201\", function () {",
+											" pm.response.to.have.status(201);",
+											"});",
+											"",
+											"pm.test(\"response validates against schema\", function() {",
+											" const schemaString = pm.collectionVariables.get(\"responseSchema201CredentialsIssue\");",
+											" pm.response.to.have.jsonSchema(JSON.parse(schemaString));",
+											"});",
+											"",
+											"// Current OpenAPI v3.x schema cannot require specific array contents",
+											"pm.test(\"response @context contains traceability/v1\", function() {",
+											" const context = pm.response.json().verifiableCredential[\"@context\"];",
+											" pm.expect(context).to.contain(\"https://w3id.org/traceability/v1\");",
+											"});",
+											"",
+											"pm.test(\"response verifiableCredential.id matches request credential.id\", function() {",
+											" const { id } = pm.response.json().verifiableCredential.id;",
+											" pm.expect(id).to.equal(pm.variables.get(\"id\"))",
```

Comment:
IMO this is the only test that should be present for this case... it is a mistake to repeat tests everywhere.